### PR TITLE
APIO-128 Fix some errors introduced in the last PR

### PR DESCRIPTION
**Pull request by ahdezma**

### CODE REVIEW

**Comment by ahdezma** [comment]
@ZoltanTakacs fix for https: https://github.com/liferay/com-liferay-apio-architect/pull/164/commits/db17d0403682f1132090da3420bfff855802ba6c

**Comment by ahdezma** [comment]
@ZoltanTakacs fix for ids: https://github.com/liferay/com-liferay-apio-architect/pull/164/commits/7c974674d8e1fff040ac8a3015432fe38ce98274

**Comment by ahdezma** [comment]
ci:test

File: apio-architect-sample/src/main/java/com/liferay/apio/architect/sample/internal/model/BlogPostingCommentModel.java [hunk 1]
```diff
@@ -72,6 +72,10 @@ public static BlogPostingCommentModel addBlogPostingComment(
 	 * @review
 	 */
 	public static void computeBlogPostingCommentModels() {
+		if (!_blogPostingComments.isEmpty()) {
```
Comment (nhpatt) [inline apio-architect-sample/src/main/java/com/liferay/apio/architect/sample/internal/model/BlogPostingCommentModel.java]: :'(

**Comment by ZoltanTakacs** [comment]
Thanks @ahdezma ! :) 👍

**Pull request merged**